<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::ObjectVisualizerWidget</class>
 <widget class="QWidget" name="GammaRay::ObjectVisualizerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="connectionSearchLine">
           <property name="placeholderText">
            <string>Filter...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pauseButton">
           <property name="text">
            <string>Pause</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="clearButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="GammaRay::DeferredTreeView" name="connectionTreeView"/>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="GvContainer" name="gvTab">
       <attribute name="title">
        <string>2D</string>
       </attribute>
      </widget>
      <widget class="VtkContainer" name="vtkTab">
       <attribute name="title">
        <string>3D</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="connTypesTab">
       <attribute name="title">
        <string>Connections</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="toolbarLayout">
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="typeSearchLine">
            <property name="placeholderText">
             <string>Filter...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="recordAllButton">
            <property name="text">
             <string>Record All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="recordNoneButton">
            <property name="text">
             <string>Record None</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="showAllButton">
            <property name="text">
             <string>Show All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="showNoneButton">
            <property name="text">
             <string>Show None</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="GammaRay::DeferredTreeView" name="typeTreeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>10</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="threadTab">
       <attribute name="title">
        <string>Threads</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="toolbarLayout_2">
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="threadSearchLine">
            <property name="placeholderText">
             <string>Filter...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="recordAllButton_2">
            <property name="text">
             <string>Record All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="recordNoneButton_2">
            <property name="text">
             <string>Record None</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="showAllButton_2">
            <property name="text">
             <string>Show All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="showNoneButton_2">
            <property name="text">
             <string>Show None</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="GammaRay::DeferredTreeView" name="threadTreeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="classTab">
       <attribute name="title">
        <string>Classes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="toolbarLayout_3">
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="classSearchLine">
            <property name="placeholderText">
             <string>Filter...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="recordAllButton_3">
            <property name="text">
             <string>Record All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="recordNoneButton_3">
            <property name="text">
             <string>Record None</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="showAllButton_3">
            <property name="text">
             <string>Show All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="showNoneButton_3">
            <property name="text">
             <string>Show None</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="GammaRay::DeferredTreeView" name="classTreeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="objectTab">
       <attribute name="title">
        <string>Objects</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="toolbarLayout_4">
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="objectSearchLine">
            <property name="placeholderText">
             <string>Filter...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="recordAllButton_4">
            <property name="text">
             <string>Record All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="recordNoneButton_4">
            <property name="text">
             <string>Record None</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="showAllButton_4">
            <property name="text">
             <string>Show All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="showNoneButton_4">
            <property name="text">
             <string>Show None</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="GammaRay::DeferredTreeView" name="objectTreeView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::DeferredTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">ui/deferredtreeview.h</header>
  </customwidget>
  <customwidget>
   <class>GvContainer</class>
   <extends>QWidget</extends>
   <header>gvcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VtkContainer</class>
   <extends>QWidget</extends>
   <header>vtkcontainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>connectionSearchLine</tabstop>
  <tabstop>pauseButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>connectionTreeView</tabstop>
  <tabstop>typeSearchLine</tabstop>
  <tabstop>recordAllButton</tabstop>
  <tabstop>recordNoneButton</tabstop>
  <tabstop>showAllButton</tabstop>
  <tabstop>showNoneButton</tabstop>
  <tabstop>typeTreeView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
